src/mixins/text.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+    
+    props: {
+
+        /**
+         *  Override vue-select default texts. 
+         *  By default vue-select will use theses values  
+         *  
+         *  noResults : "Sorry no matching options."
+         *  loading : "Loading..."
+         *  searchOption : "Search for options"
+         *  clearSelection : "Clear selection"
+         *  
+         *  @type {Object}
+         */
+        texts : {
+            type: Object, 
+            default : () => ({}),
```

Comment:
I think it would be nice to actually provide the default props here. The current solution can be misleading when debugging with the vue-devtools by instance, `texts` prop magically has default values the user does not see unless diving in the source here.

Suggestion might be to put the variable `defaultTexts` outside of the scope (top of the file as a const), then reference it both in the default function initially, and in the return statement of the `textValues` computed function.

IMO the `textValues `function is not really needed due to the fact that the user can simply spread in their custom props themselves, but ey 👈😎👉 it's also chill not to do it yourself.

Reply:
<img src="https://i.kym-cdn.com/photos/images/newsfeed/001/878/329/dfa.jpg" height="100px">